README.md

```diff
@@ -59,13 +59,15 @@ dsc_windowsfeature {'IIS':
 }
 ~~~
```

Comment:
@jpogran spot check?

would be the outcome of this native DSC Resource:

~~~powershell
WindowsFeature IIS {
  Ensure = 'present'
  Name   = 'Web-Server'
}
~~~